examples/cloudwatch_listener_ex.py

```diff
@@ -0,0 +1,17 @@
+from locust import main, events
+from locust_plugins.listeners.cloudwatch import CloudwatchAdapter, ServiceContext
+from locust_plugins import missing_extra
+
+try:
+    import boto3
+except ModuleNotFoundError:
```

Comment:
This check is meant to be in the plugin itself, not in the example code

Reply:
The plug-in is written in such a way that it can work with any aws wrapper which supports the `put_metric_data` method required. That is why I did not add the dependency inside the plug-in. 
Also boto initialisation might vary from one setup to another (different configurations are supported). So did not want to make that hardcoded in the plug-in.
Please do let me know if you want me to change the same.

Reply:
I see. You can do the try-catch import of boto3 inside cloudwatch.py instead. I'd prefer if users configured boto3 using standard env vars instead of from code, because it simplifies the interface. If you want, you could leave it as an option, but I'd like the default to be that boto3.client is created in cloudwatch.py.

Reply:
I have done the changes as much as I understand it. Please take a look